crates/nu-command/src/filters/find.rs

```diff
@@ -47,12 +47,17 @@ impl Command for Find {
             .switch("invert", "invert the match", Some('v'))
             .rest("rest", SyntaxShape::Any, "terms to search")
             .category(Category::Filters)
+            .extra_usage("Search terms: filter")
```

Comment:
Do we need this? Previously all we had to do was add the `fn extra_usage()`

Reply:
I think the `filter` search term here is redundant since it is also the command category.